Configurations/make-release-package.sh

```diff
@@ -52,17 +52,30 @@ if [ "$ACTION" = "" ] ; then
     cd "$CONFIGURATION_BUILD_DIR/staging-spm"
     #rm -rf "$CONFIGURATION_BUILD_DIR/Sparkle.xcarchive"
     zip -rqyX -9 "../Sparkle-for-Swift-Package-Manager.zip" *
+    
+    # GitHub actions set the CI environment variable to true
+    if [ "$CI" != true ]; then
+        # Get latest git tag
+        cd "$SRCROOT"
+        latest_git_tag=$(git describe --abbrev=0)
+        # Check semantic versioning
+        [[ $latest_git_tag =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-((0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?(\\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?$ ]] && echo "Tag $latest_git_tag follows semantic versioning" || echo "WARNING: Tag $latest_git_tag does not follow semantic versioning! SPM will not be able to resolve the repository"
```

Comment:
This line checks whether the latest tag follows semantic versioning and outputs a warning if it doesn't. Should this be an error instead?

Reply:
Yeah, it should be an error.

Reply:
Ok, it's now an error!